lib/chat/ui/widgets/chat_list.dart

```diff
@@ -82,12 +92,18 @@ class ChatList extends StatelessWidget {
     switch (message.messageType) {
       case MessageType.sent:
         return MessageBubbleStyles.buildStyleSent();
-      case MessageType.received:
-        return MessageBubbleStyles.buildStyleReceived();
       case MessageType.header:
         return MessageBubbleStyles.buildStyleHeader();
       case MessageType.loading:
         return MessageBubbleStyles.buildStyleLoading();
+      case MessageType.received:
+        return MessageBubbleStyles.buildStyleReceived();
+      case MessageType.receivedStackTop:
```

Comment:
don't like using type for the presentation logic. this will need to be addressed in refactor.

Reply:
Sure, will change it in refactor then.